src/ext/exe.rs

```diff
@@ -269,47 +253,48 @@ impl Exe {
 }
 
 /// Tailwind uses the 'vMaj.Min.Pat' format.
-/// WASM opt uses 'version_NNN' format.
-/// Cargo-generate has the 'vX.Y.Z' format
 /// We generally want to keep the suffix intact,
 /// as it carries classifiers, etc, but strip non-ascii
 /// digits from the prefix.
 #[inline]
-fn sanitize_version_prefix(ver_string: &str) -> String {
-    ver_string
-        .chars()
-        .skip_while(|c| !c.is_ascii_digit() || *c == '_')
-        .collect::<String>()
+fn sanitize_version_prefix<'a>(ver_string: &'a str) -> Result<&'a str> {
```

Comment:
Since `wasm-opt` is now used as a library instead of a binary downloaded from Github, I figured the check for "version_" is unnecessary.